src/c++/library/http_client.h

```diff
@@ -262,7 +262,8 @@ class InferenceServerHttpClient : public InferenceServerClient {
   /// loading the model.
   /// \param encoded_files Optional map specifying file path (with "file:"
   /// prefix) in the override model directory to the base64 encoded file
-  /// content. If specified, 'config' must be provided to be
+  /// content. The files will form the model directory that the model
+  /// to be loaded from. If specified, 'config' must be provided to be
```

Comment:
The files will form the model directory that the model ~-to-~ **will** be loaded from?